web3/ethcallsigs.nim

```diff
@@ -24,7 +24,7 @@ proc eth_getBlockTransactionCountByNumber(blockId: BlockIdentifier)
 proc eth_getUncleCountByBlockHash(data: BlockHash)
 proc eth_getUncleCountByBlockNumber(blockId: BlockIdentifier)
 proc eth_getCode(data: Address, blockId: BlockIdentifier): seq[byte]
-proc eth_sign(data: Address, message: seq[byte]): seq[byte]
+proc eth_sign(address: Address, data: string): seq[byte]
```

Comment:
Looks like the return value should also be a hex string according to that spec.

I guess this is the better version https://github.com/status-im/nimbus-eth1/blob/ef7773daa6af7cf71916c31526b1257943498754/tests/rpcclient/ethcallsigs.nim ? There are some other `seq[byte]` vs string differences with that version, not sure as to why..

Reply:
Hmm, it does seem to convert the result into a byte sequence now, which is quite convenient.